doc/tutorial/Client.lhs

```diff
@@ -93,6 +93,18 @@ position :<|> hello :<|> marketing = client api
 
 `client api` returns client functions for our _entire_ API, combined with `:<|>`, which we can pattern match on as above. You could say `client` "calculates" the correct type and number of client functions for the API type it is given (via a `Proxy`), as well as their implementations.
 
+If there is an `EmptyAPI` within your API, this matches the `EmptyClient`
```

Comment:
I think it might be nice to clarify things a bit more here as well. Something like:

> If you have an `EmptyAPI` in your API, servant-client will hand you a value of type `EmptyClient` in the corresponding slot, where `data EmptyClient = EmptyClient`, as a way to indicate that you can't do anything useful with it.

?